backend/kesaseteli/applications/models.py

```diff
@@ -675,17 +675,17 @@ def is_applicant_dead_according_to_vtj(self) -> bool:
         )
 
     @property
-    def vtj_last_name(self):
+    def vtj_last_name(self) -> Optional[str]:
```

Comment:
Looks for me that the return value was previously optional string but now it's just string because `None` was replaced with "". Or could `vtj_last_names[0]` still return `None`?

Reply:
Also the [documentation](https://docs.python.org/3/library/typing.html#typing.Optional) says that Optional[str] could be also expressed as `str | None` which is at least for me a lot cleaner

Reply:
> Looks for me that the return value was previously optional string but now it's just string because `None` was replaced with "". Or could `vtj_last_names[0]` still return `None`?

I left the Optional there exactly because I was not 100% sure whether vt_last_names[0] will always return a string or could it possibly return None.